rails-commenting.rb

```diff
@@ -55,14 +62,17 @@ def destroy
       redirect_to blog_posts_path
     else
       # ---8)
+      # it gives direction to a certain page. in this case (blog_post_path(@post)) will go to the blog post page.
       redirect_to blog_post_path(@post)
     end
   end
 
   # ---9)
+  # private keyword can only be used when called.
```

Comment:
Private makes it so any method below it can only be called from within this class.